support/cas-server-support-ldap-core/src/main/java/org/apereo/cas/authentication/support/DefaultAccountStateHandler.java

```diff
@@ -156,14 +156,16 @@ protected void handleWarning(
             return;
         }
 
-        final ZonedDateTime expDate = DateTimeUtils.zonedDateTimeOf(warning.getExpiration());
-        final long ttl = ZonedDateTime.now(ZoneOffset.UTC).until(expDate, ChronoUnit.DAYS);
-        LOGGER.debug(
+        if(warning.getExpiration()!=null){
```

Comment:
I may also suggest perhaps adding a log message in either scenario, explaining "the expiration found is X" or "could not find an expiration", etc.